CleverPush/Source/CleverPushInstance.m

```diff
@@ -1205,9 +1208,20 @@ - (void)handleSubscription:(CPHandleSubscribedBlock _Nullable)subscribedBlock fa
     } failure:failureBlock skipTopicsDialog:skipTopicsDialog];
 }
 
-- (void)requestNotificationPermission:(void (^)(BOOL granted, NSError* error))completionHandler {
+- (void)requestNotificationPermission:(BOOL)shouldShowAlert playSound:(BOOL)shouldPlaySound setBadge:(BOOL)shouldSetBadge
```

Comment:
Isn't this method used by customers as well?
Should we keep the old method with default settings so no breaking changes will be there?

Reply:
@gbhrdt ,
This method `requestNotificationPermission`  is not public means the customer will not be able to use this.